package.json

```diff
@@ -4,71 +4,75 @@
     "scripts": {
         "help:help": "echo Prints a list of available scripts and a reminder about how to get more help.",
         "help": "echo Available scripts -- execute \\`npm run help:SCRIPTNAME\\` for help on SCRIPTNAME: && (npm run | grep '^\\s\\s\\S' | grep -v 'help\\|only\\|postinstall\\|prepare')",
-        "help:dev": "run-s -s help:dev:*",
-        "help:dev:1": "echo Compiles frontscope and starts a server running it.",
-        "help:dev:2": "echo Note the server will hot-reload the code from any",
-        "help:dev:3": "echo source files as they change, to ease development.",
+        "help:dev": "run-s -s help:dev:[1-3]",
+        "help:dev:1": "echo \"Compiles frontscope and starts a server running it.\"",
+        "help:dev:2": "echo \"Note the server will hot-reload the code from any\"",
+        "help:dev:3": "echo \"source files as they change, to ease development.\"",
+        "help:dev:workbench": "run-s -s help:dev:workbench:[1-3]",
```

Comment:
No, actually the `help:dev:workbench:*` is better, because then there is only one source of truth: the number of help:dev:workbench messages that actually exist. If you do that, then a future maintainer who has to add another line of help:dev:workbench description doesn't have to redundantly update the number of messages in this `run-s` pattern. The perfect coding ideal is just one source of truth for each piece of information about a program. We generally won't quite get there, but when we have an alternative, choose the one that duplicates information the least.

So as long as we are at it, why don't you put the help messages for just `dev` in, say, `help:dev::1` through `help:dev::3` so that the `run-s` pattern for `help:dev` can be just `help:dev::*`. That will put all of the little help commands back to being uniform in terms of their structure. Thanks!

Reply:
These fine points in the help: scripts of package.json still need to be addressed.

Reply:
> The perfect coding ideal is just one source of truth for each piece of information about a program

Good point. Done!